### PR TITLE
Create gizmo.py

### PR DESCRIPTION
**Pull request by kunalghosh**
Challenge 1

### CODE REVIEW

File: gizmo.py [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# Test gizmo file
+def hello(name, country='Finland'):
+  print('Hello {name}, how are things in {country}?')
+  
+def spell(name):
+  total_chars = len(name)
+  for idx,char in enumerate(name):
+    print(char, end="")
+    if idx != total_chars-1:
+      print(".", end="")
```
Comment (wmvanvliet) [inline gizmo.py]: We still need a newline at the very end :)

File: gizmo.py [hunk 2]
```diff
@@ -0,0 +1,10 @@
+# Test gizmo file
+def hello(name, country='Finland'):
+  print('Hello {name}, how are things in {country}?')
```
Comment (wmvanvliet) [inline gizmo.py]: you forgot the `f` at the beginning of the string

File: gizmo.py (hunk 1 above)
Comment (kunalghosh) [reply gizmo.py]: Fixed it :)

File: gizmo.py (hunk 2 above)
Comment (kunalghosh) [reply gizmo.py]: Ah yes, didn't see that. Fixed now 👍🏽

**Comment by kunalghosh** [comment]
Would you be going through a bit about how to write `git actions` ? Would be really fun to learn a bit more about them.

**Comment by wmvanvliet** [comment]
Unfortunately we won't be covering how to do specific technical tasks. This course will deal with how to organize and design code. To learn how to setup git actions, I would refer you to the excellent [documentation](https://docs.github.com/en/actions) on the subject, and of course you are welcome to have a look at how the actions are set up for this repo: https://github.com/wmvanvliet/gizmo/blob/master/.github/workflows/test_answers.yml

**Comment by wmvanvliet** [comment]
THe theory session started in this zoom room, btw, come join us! https://aalto.zoom.us/j/69625782291?pwd=UW0zV24yYy9LWVlITEhTWGFDMmptdz09

**Pull request closed without merging**